src/mainwindow.ui

```diff
@@ -117,10 +117,17 @@
     <addaction name="separator"/>
     <addaction name="action_anim_delete"/>
    </widget>
+   <widget class="QMenu" name="menuUtilities">
+    <property name="title">
+     <string>Utilities</string>
```

Comment:
Ok, knowing this creating a Utilities menu looks fine for me.

> Btw. What is your release planning? I'm not involved in the Trove community and I have no idea if anyone or who is using Zoxel.

The plan was to get an 0.5.15 out before christmas and if necessary a 0.5.16 afterwards so that the 0.6 update would only contain to (hopefully not for to many people) breaking render upgrade code.
According to the Github releases api the v0.5.13 (which was out for quite a while) was downloaded 1496 times for windows and 471 for os x.